### PR TITLE
Lvjav442 7

### PR DESCRIPTION
**Pull request by nastiamazurak**

### CODE REVIEW

**Review by martavoloshyn — DISMISSED** [review]

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Role.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.softserve.ita.java442.cityDonut.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Role.java:8]: delete row

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Role.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "role", length = 25, nullable = false)
+    private String role;
+
+}
+
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Role.java]: delete rows

File: src/main/java/com/softserve/ita/java442/cityDonut/repository/RoleRepository.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.softserve.ita.java442.cityDonut.repository;
+
+
+import com.softserve.ita.java442.cityDonut.model.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.Optional;
+
+public interface RoleRepository extends JpaRepository {
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/repository/RoleRepository.java]: JpaRepostory has Generic

File: src/main/java/com/softserve/ita/java442/cityDonut/repository/RoleRepository.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package com.softserve.ita.java442.cityDonut.repository;
+
+import com.softserve.ita.java442.cityDonut.model.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/repository/RoleRepository.java:8]: add @Repository

**Review by NazarRyvak — APPROVED** [review]

**Review by martaprotsak — APPROVED** [review]

**Pull request merged**